src/open_inwoner/userfeed/tests/hooks/test_case_status.py

```diff
@@ -141,37 +141,51 @@ def test_status_update_with_status_type_config(
             catalogus__url=case.zaaktype.catalogus,
             catalogus__service=self.api_group.ztc_service,
         )
-        status_config = ZaakTypeStatusTypeConfigFactory(
+        ZaakTypeStatusTypeConfigFactory(
             zaaktype_config=ztc,
             statustype_url=status.statustype.url,
             status_indicator=StatusIndicators.warning,
             status_indicator_text="my_status_text",
             action_required=True,
         )
 
-        case_status_notification_received(user, case, status)
-
-        # check feed
-        feed = get_feed(user)
-        self.assertEqual(feed.total_items, 1)
-        self.assertEqual(len(feed.summary), 2)
+        scenarios = [
+            (ZaakTitleDisplayChoices.zaak_omschrijving, case.omschrijving),
+            (ZaakTitleDisplayChoices.zaaktype_omschrijving, case.zaaktype.omschrijving),
+            (ZaakTitleDisplayChoices.zaaktype_onderwerp, case.zaaktype.onderwerp),
+        ]
+        for zaak_title_config, expected_zaak_title in scenarios:
+            with self.subTest(f"zaak_title_config: {zaak_title_config}"):
+                self.api_group.open_zaak_config.derive_zaak_titel_from = (
+                    zaak_title_config
+                )
```

Comment:
No `.save()` needed?
```suggestion
                self.api_group.open_zaak_config.derive_zaak_titel_from = (
                    zaak_title_config
                )
                self.api_group.open_zaak_config.save()
```

Reply:
Yep, already fixed

Reply:
It was indeed taking the title from `zaaktype.omschrijving` for all cases, so tests passed accidentally. Subtests were a good idea therefore.